spec/support/reporters/test-coverage.mjs

```diff
@@ -8,11 +8,14 @@ export default class TestCoverageReporter {
   }
 
   jasmineDone() {
-    const thisFileDirname = path.dirname(new URL(import.meta.url).pathname)
-    const nycOutputPath = path.join(thisFileDirname, '..', '..', '..', '.nyc_output');
+    const thisFileDirname = path.dirname(new URL(import.meta.url).pathname);
+    const rootDirname = path.join(thisFileDirname, '..', '..', '..');
```

Comment:
I'm wondering if there's another way, maybe via Node, to get this root directory name. I'm just thinking that if this file gets moved, this path won't work anymore.